docker/README.md

```diff
@@ -1,23 +1,41 @@
+# Introduction
+
+This README describes two ways to deploy the containers for web UI and the workers.
+
+The first one describes how to deploy a openQA environment using docker with
+the fedoraqa images or images built locally.
+
+The second one uses a docker-compose to deploy a complete web UI with HA for the
+API and the dashboard.
+
+If a section or action is tagged as "Fedora", it shall be applied only for
+Fedora deployment. On the contrary, if the tag "docker-compose" is used it will
+be used for the Docker-compose deployment.
```

Comment:
```suggestion
The first one describes how to deploy an openQA environment using docker with
the Fedora images or images built locally.

The second one uses a docker-compose to deploy a complete web UI with HA for the
API and the dashboard.

If a section or action is tagged as "Fedora", it shall be applied only for
Fedora deployment. On the contrary, if the tag "docker-compose" is used it will
be used for the docker-compose deployment.
```

Reply:
Thanks. Done